jsk_2013_04_pr2_610/README.md

```diff
@@ -1,6 +1,8 @@
 jsk_2013_04_pr2_610
 ====================
 
+[![IRT demo](http://img.youtube.com/vi/ToL3egTOahg/0.jpg)](https://www.youtube.com/watch?v=ToL3egTOahg "IRT demo")
+
```

Comment:
please add a comment on what is this video and how this related to this repository

Reply:
Moved the location of the video.
There is already description about the relation between the demo and IRT demo.
Should I add more details?